docs/shared-content/infrastructure/create-deployment-targets-warning.include.md

```diff
@@ -0,0 +1,3 @@
+:::warning
+Package acquisition is not performed automatically after a deployment target is dynamically created from scripts. If following steps require packages to be acquired on these new deployment targets, make sure you add a [health check](/docs/projects/built-in-step-templates/health-check.md) step for the corresponding target roles after the target creation step. This will ensure required packages are downloaded for the new targets.
```

Comment:
Reads well and is to the point 👍.

`If following steps require packages` could change a little, maybe `If subsequent steps require packages` but that's a personal preference and I'm happy either way 🙂